### ISSUE TITLE
Plugin not working with comicpress

### ISSUE THREAD

**Issue by Lumaga** [WPGlobus/wpglobus-featured-images#1]
Hello!
I'm running a webcomic at http://fourleafcomic.com/, and wpglobus featured images is exactly what I need to make my comic bilingual, because the theme comicpress (and the plugin comic easel) uses featured images to publish the comic pages. But the plugin wpglobus featured images doesn't seem to work with that theme. Is there a way to fix that? I'd be very grateful.
Thanks for your time!

**Comment by alexgff** [comment]
Are you using  WPGlobus featured images  with https://wordpress.org/plugins/wpglobus/ ?

**Comment by Lumaga** [comment]
Hi Alex, thank you for replying.
Yes, I'm using WPGlobus, I saw it was necessary for WPGlobus Featured Images to work, and also I'm using it to translate the rest of my content.

**Comment by alexgff** [comment]
I checked WPGlobus+WPGlobus FI + ComicPress on my test site. They work well. 
Try deactivate another plugins and find what interfere with WPGLobus.

**Comment by Lumaga** [comment]
I tryed deactivating all my plugins, one by one, except for WPGlobus and
WPGlobus Featured Images but the problem persists.
I thought it may have something to do with the fact that I'm using a child
theme of ComicPress, but I changed to the parent theme and nothing.
Just in case, the plugins that I'm using are Akismet, Comic Easel, Jetpack,
WP Super Cache, WPGlobus and WPGlobus Featured Images.

2016-10-16 21:06 GMT-03:00 Alex Gor <notifications@github.com>:
